scripts/prepare-preview-builds.sh

```diff
@@ -30,10 +30,28 @@ prepare-preview-manifest() {
 }
 
 echo "Preparing manifests..."
+# First, update the root package.json
+echo "- root package.json"
+prepare-preview-manifest "package.json"
+
+# Then update all workspace package.json files
 while IFS=$'\t' read -r location name; do
   echo "- $name"
   prepare-preview-manifest "$location/package.json"
 done < <(yarn workspaces list --json | jq --slurp --raw-output 'map(select(.location != ".")) | map([.location, .name]) | map(@tsv) | .[]')
 
 echo "Installing dependencies..."
 yarn install --no-immutable
+
+echo "Updating TypeScript imports for React Native packages..."
+for pkg in "design-system-react-native" "design-system-twrnc-preset"; do
+  if [ -d "packages/$pkg/src" ]; then
+    echo "- $pkg source files"
+    # Use different sed syntax for Linux vs macOS
+    if [[ "$OSTYPE" == "darwin"* ]]; then
+      find "packages/$pkg/src" -type f -name "*.ts*" -exec sed -i '' "s/@metamask\//@metamask-previews\//g" {} +
+    else
+      find "packages/$pkg/src" -type f -name "*.ts*" -exec sed -i "s/@metamask\//@metamask-previews\//g" {} +
+    fi
+  fi
+done
```

Comment:
Since we're using `tsc` for the React Native packages instead of `ts-bridge`, we need to update the TypeScript imports in source files to match the preview package scope. 

The React packages use `ts-bridge` which handles import resolution differently, but for `design-system-react-native` and `design-system-twrnc-preset`, TypeScript needs the import statements to exactly match the package names in package.json: